platforms/documentation/docs/src/snippets/bestPractices/useCacheabilityAnnotations/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+// tag::avoid-this[]
+tasks.register<Delete>("clean") {
+    delete(layout.buildDirectory)
+}
+
+abstract class BadCalculatorTask : DefaultTask() { // <1>
+    @get:Input
+    abstract val first: Property<Int>
+
+    @get:Input
+    abstract val second: Property<Int>
+
+    @get:OutputFile
+    abstract val outputFile: RegularFileProperty
+
+    @TaskAction
+    fun run() {
+        val result = (first.get() + second.get()).toString()
```

Comment:
Suggestion to make it look cleaner:

```
        val result = first.get() + second.get()
        logger.lifecycle("Result: $result")
        outputFile.get().asFile.writeText(result.toString())
```

Reply:
Ah, whoops.  I updated the Groovy but not the Kotlin.  Good catch.